src/neuroconv/datainterfaces/ophys/baseimagingextractorinterface.py

```diff
@@ -127,14 +148,15 @@ def get_metadata(
                     two_photon_series["rate"] = float(two_photon_series["rate"])
         return metadata
 
-    def get_original_timestamps(self) -> np.ndarray:
+    def get_original_timestamps(self) -> np.ndarray:  # noqa: D102
         reinitialized_extractor = self.get_extractor()(**self.source_data)
         return reinitialized_extractor.frame_to_time(frames=np.arange(stop=reinitialized_extractor.get_num_frames()))
 
-    def get_timestamps(self) -> np.ndarray:
+    def get_timestamps(self) -> np.ndarray:  # noqa: D102
         return self.imaging_extractor.frame_to_time(frames=np.arange(stop=self.imaging_extractor.get_num_frames()))
 
     def set_aligned_timestamps(self, aligned_timestamps: np.ndarray):
+        """Replace all timestamps for this interface with those aligned to the common session start time."""
```

Comment:
This needs a Parameters section for aligned_timestamps

Reply:
Thanks.

Reply:
Changed to noqa, this docstring is identical and should be inherited from the base class.